### PR TITLE
adding updated jsx and css files for About section

### PR DESCRIPTION
**Pull request by dombrown95**
Basic About Us jumbotron with placeholder background image and provided text - 

Screenshot 2024-01-18 141025

### CODE REVIEW

File: src/components/AboutUs/index.jsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-    <div className="about-us-container">
-      <h1>About Us</h1>
+    <div className="jumbotron jumbotron-fluid">
+      <div className="container">
+        <h2 className="display-4">About Us</h2>
+        <p className="lead">At QuiglyCare, we understand the importance of compassionate and reliable care for your 
+loved ones. With our comprehensive range of services, we provide a nurturing and 
+supportive environment that enhances their well-being. Explore the possibilities with 
+QuiglyCare and experience personalised care tailored to your unique needs.</p>
+      </div>
```
Comment (ewelinasobora) [inline src/components/AboutUs/index.jsx:14]: Could you please convert all the text in the components to JSON format within src/data?

**Review by ewelinasobora — CHANGES_REQUESTED** [review]

File: src/components/AboutUs/index.jsx (hunk 1 above)
Comment (dombrown95) [reply src/components/AboutUs/index.jsx]: Hi @ewelinasobora I've made the requested changes for when you have a minute to take a look, thanks

Comment (ewelinasobora) [reply src/components/AboutUs/index.jsx]: yep this looks good

File: src/components/AboutUs/index.jsx [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 function AboutUs() {
   return (
-    <div className="about-us-container">
-      <h1>About Us</h1>
+    <div className="jumbotron jumbotron-fluid">
+      <div className="container">
+        <h2 className="display-4">{aboutUsData.title}</h2>
+        <p className="lead">{aboutUsData.content}</p>
+      </div>
     </div>
```
Comment (ewelinasobora) [inline src/components/AboutUs/index.jsx:13]: could you use react-bootstrap jumbotron componen, please? https://react-bootstrap-v3.netlify.app/components/jumbotron/

Comment (ewelinasobora) [reply src/components/AboutUs/index.jsx:13]: you can also check container component if it fits

File: src/components/AboutUs/index.jsx (hunk 1 above)
Comment (dombrown95) [reply src/components/AboutUs/index.jsx]: Hi @ewelinasobora I've added the additional placeholder images below the jumbotron - 

Screenshot 2024-01-18 204450

File: src/components/AboutUs/index.jsx (hunk 2 above)
Comment (dombrown95) [reply src/components/AboutUs/index.jsx]: done @ewelinasobora

Comment (ewelinasobora) [reply src/components/AboutUs/index.jsx]: looks good

**Pull request merged**